src/pages/Vote/DelegateModal/index.tsx

```diff
@@ -0,0 +1,63 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Formik } from 'formik';
+import { Typography } from '@mui/material';
+import { Modal, NoticeInfo, FormikSubmitButton, FormikTextField, TextButton } from 'components';
+import Path from 'constants/path';
+import { useTranslation } from 'translation';
+import addressValidationSchema from './addressValidationSchema';
+import { useStyles } from './styles';
+
+interface IDelegateModalProps {
+  onClose: () => void;
+  open: boolean;
+  address: string;
```

Comment:
Nit: I'd rename this something like `currentUserAccountAddress` to make it obvious what it is